docs/3.0.0-beta.x/guides/external-data.md

```diff
@@ -0,0 +1,96 @@
+# Fetching external data
+
+This guide will explain how to fetch data in an external service and use it in your app.
+
+In this example we will see how to daily fetch Docker pull count and store it in database.
```

Comment:
In this example we will see how to daily fetch Docker pull count to store the result in your database.